core/lib/vm/src/tracers/default_tracers.rs

```diff
@@ -41,6 +40,7 @@ pub(crate) struct DefaultExecutionTracer<S, H: HistoryMode> {
     in_account_validation: bool,
     final_batch_info_requested: bool,
     pub(crate) result_tracer: ResultTracer,
+    pub(crate) refund_tracer: Option<RefundsTracer>,
```

Comment:
can you add more comment here (and explain why the RefundsTracer is needed here at all - and it cannot be simply in the list of custom tracers)